www/src/components/layout.js

```diff
@@ -154,7 +154,12 @@ class DefaultLayout extends React.Component {
 
     return (
       <>
-        <SiteMetadata pathname={this.props.location.pathname} />
+        <SiteMetadata
+          pathname={this.props.location.pathname}
+          metaDescription={this.props.metaDescription}
```

Comment:
Few comments:

I'd either use a `meta` array as a prop _or_ remove the prefixes (e.g. because it's SiteMetadata it should be obvious that these will eventually become metatags).

In other words:

```jsx
<SiteMetadata
  pathname={this.props.location.pathname}
  description={this.props.description}
  image={this.props.image}
/>
```

Reply:
Agreed. I think I originally had it without prefixes.